src/App.js

```diff
@@ -1,13 +1,61 @@
-import React from "react";
 import "./App.css";
+import Header from './components/Header';
+import PictureCard from './components/PictureCard';
+import Discover from './components/Discover';
+import PreviousDates from './components/PreviousDates';
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import moment from "moment";
 
 function App() {
-  return (
+
+  function formatDate() {
+    let today = new Date();
+    let todayStr = moment(today).format('YYYY-MM-DD');
+
+    return todayStr;
+  }
+
+  const [photoState, setPhotoState] = useState();
```

Comment:
No initial value for `photoState`? maybe null? `const [photoState, setPhotoState] = useState(null);`